src/app/components/Search/Search.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import SearchForm from './SearchForm';
+import SearchResultsList from './SearchResultsList';
+import * as searchActions from '../../actions/SearchActions';
+
+class Search extends React.Component {
```

Comment:
From the architecture of this app and component, I think maybe `SearchContainer` would be a better name for this container. I was expecting to just find the search form in `Search` but it turns out to be the layout for the main page.

Reply:
Will rename. Wasn't sure if this was actually going to work and wasn't sure how specific to be in naming for container, component, etc.